pages/platform/databases/databases_12_operator/guide.en-gb.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Getting started with database kubernetes opdrator
+excerpt: Find out how to install and use the kubernetes database operator 
+slug: database-operator
+section: General guides
+order: 120
+updated: 2023-05-02
+---
+
+**Last updated May 2<sup>nd</sup>, 2023**
+
+## Objective
+
+The kubernetes database operator allow you to automaticaly authorize your Kubernetes cluster IP on your OVHcloud cloud databases service.
+
+**This guide explains how to install and use the database operator in order to authorize the Kubernetes IP on your service**
+
+## Requirements
+
+- Access to the [OVHcloud API](https://api.ovh.com/) (create your credentials by consulting [this guide](https://docs.ovh.com/gb/en/api/first-steps-with-ovh-api/))
+- A [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+
+## Ovh Credentials
+The operator needs a secret that contains the credentials to call Ovhcloud api. Go to https://api.ovh.com/createToken/ to generate the credentials namely:
+- application key
+- application secret
+- consumer key
+
+## Values
+Create a values.yaml to be injected in the helm chart 
+that will be created afterwards. Region is either: ovh-eu, ovh-ca or ovh-us.
+```yaml
+ovhCredentials:
+  applicationKey: XXXX
+  applicationSecret: XXXX
+  consumerKey: XXXX
+  region: XXXX # ovh-eu, ovh-ca or ovh-us
+
+namespace: XXXX # Your Kubernetes namespace
```

Comment:
It's not a best practice to define the namespace in the Helm chart values.

Instead, you can specify `--namespace` directly in the `helm install` command line.